src/common/components/entry-menu/index.tsx

```diff
@@ -300,21 +378,78 @@ export class EntryMenu extends BaseComponent<Props, State> {
             ];
         }
 
-        if (this.canPinOrMute()) {
+        if (this.canPinBothOptions()) {
```

Comment:
@feruzm can you textually elaborate this behaviour or is it possible to test it using demo.com, any community I can post to and this test this as moderator etc...

What I understand is, this is a special case when post is in community and author is also moderator trying to pin post.

Reply:
@noumantahir Yes, let's say @demo.com is moderator of Ecency community and makes post in that community. So he can potentially pin that post to Ecency community as well as he can pin to his own blog. He can pin any content in Ecency community published by others as well. Only his own can be pinned into his profile/blog. Web code logic is implemented in a bit complicated way for some reason, perhaps due to lack of understanding all scenarios.

Basically you can pin any of you own post into your blog. You can post any content in community page if you are moderator of that community. So unpinning also comes with those conditions.